packages/init/src/steps/install-required-dependencies.ts

```diff
@@ -19,19 +40,8 @@ export async function installRequiredDependencies(
     `${installDevDependencyCommand} ${Utils.CONSTS.SM_PACKAGE_NAME}`
   );
 
-  switch (framework) {
-    case Utils.Framework.FrameworkEnum.react:
-    case Utils.Framework.FrameworkEnum.next:
-      await execCommand(
-        `${installDependencyCommand} ${Utils.CONSTS.PRISMIC_REACT_PACKAGE_NAME}`
-      );
-      break;
-    case Utils.Framework.FrameworkEnum.svelte:
-      await execCommand(
-        `${installDependencyCommand} ${Utils.CONSTS.PRISMIC_DOM_PACKAGE_NAME}`
-      );
-      break;
-  }
+  const deps = depsForFramework(framework);
+  if (deps) await execCommand(`${installDependencyCommand} ${deps}`);
```

Comment:
I'm wondering if we should install with the `@latest` tag all the time 🤷‍♂️

Reply:
Maybe for QA it would be nice for them to have a flag to state tagged versions